FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -261,6 +265,11 @@ uint8_t ucMinAgeFound = 0U;
 					optimisation. */
 					xARPCache[ x ].ucAge = ( uint8_t ) ipconfigMAX_ARP_AGE;
 					xARPCache[ x ].ucValid = ( uint8_t ) pdTRUE;
+
+					/* MISRA rule 15.5 relaxed for reduced complexity. Not putting
+					 * a return here will increase cyclomatic complexity in later
+					 * part of the code */
+					/* coverity[misra_c_2012_rule_15_5_violation] */
```

Comment:
Do __not__ disable this rule and delete lines 268-272.  We __want__ Coverity to continue warning about this so that we remember to go back and refactor this function later.  The goal is __not__ to eliminate MISRA warnings, the goal is _to do the right thing._  And overly complex functions should be refactored.